src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,58 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 706px;
+  padding: 10px;
+  gap: $gap-size;
+  margin: 0 auto;
+
+  .calendar__day-box {
+    background-color: #eee;
+    width: $box-size;
+    height: $box-size;
+    border: 1px solid black;
+    box-sizing: border-box;
+    text-align: center;
+    line-height: $box-size;
+    transition: transform 0.5s;
```

Comment:
You should also list background color here because currently it's changing to pink immediately